Library/ViewModels/ChangePasswordViewModel.swift

```diff
@@ -128,8 +129,13 @@ ChangePasswordViewModelInputs, ChangePasswordViewModelOutputs {
         } else {
           return nil
         }
-    }.skipNil()
-    .skipRepeats()
+    }
+
+    self.validationErrorLabelMessage = errorLabelMessage
+      .skipNil()
+      .skipRepeats()
+
+    self.accessibilityFocusValidationErrorLabel = self.validationErrorLabelMessage.ignoreValues()
```

Comment:
I totally forgot to mention that we should only trigger this signal if `VoiceOver` is turned ON (for consistency with the rest of the code base)

```
.filter { AppEnvironment.current.isVoiceOverRunning() }
```